calendar.php

```diff
@@ -596,6 +596,46 @@
 		// Redirect back to the main calendar view.
 		redirect("calendar.php", $lang->redirect_eventdeleted);
 	}
+	else
+	{
+		error($lang->delete_no_checkbox);
```

Comment:
I added it to the language file: https://github.com/mybb/mybb/pull/2006/files#diff-505deb1daf1b38b7bace47d542d761e5R93 - is there anything else I need to do?